### PR TITLE
OCM-5461 | feat: add HostedControlPlaneDefault to version type model

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-5461

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.347 Jan 23 2024
```
Comment (lucasponce) [inline CHANGES.md:5]: I think the changes are typically added when the PR is going to be released.

 @ciaranRoche @gdbranco how do you want to deal with this?
 
 Keeping the change in the model in one PR, and consolidate the changes in the doc when the tag is created?

Comment (lucasponce) [reply CHANGES.md:5]: Change is good, just to not break any flow in the release process

File: model/clusters_mgmt/v1/version_type.model [hunk 2]
```diff
@@ -37,6 +37,10 @@ class Version {
 	// HostedControlPlaneEnabled indicates whether this version can be used to create HCP clusters.
 	HostedControlPlaneEnabled Boolean
 
+	// HostedControlPlaneDefault is a flag that indicates if this should be selected as the default version when a
+	// HCP cluster is created without specifying explicitly the version.
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/version_type.model:41]: Only thing I'm not sure is how the model generator behaves for 2 lines of comments. Haven't seen it before

Comment (chenz4027) [reply model/clusters_mgmt/v1/version_type.model:41]: @gdbranco Should be okay, the [EndOfLifeTimestamp](https://github.com/openshift-online/ocm-api-model/blob/084e74c4f3a0378e757e8d30d5418bd0b9c19e79/model/clusters_mgmt/v1/version_type.model#L48) also has two line of comments.

**Comment by lucasponce** [comment]
LGTM

**Review by lucasponce — APPROVED** [review]

**Pull request merged**